src/character.cpp

```diff
@@ -7281,3 +7286,86 @@ void Character::use_fire( const int quantity )
         return;
     }
 }
+
+int Character::heartrate_bpm() const
+{
+    //Dead have no heartbeat usually
+    if( is_dead_state() ) {
+        return 0;
+    }
+    //No heartbeat in omnicell
+    if( has_trait( trait_SLIMESPAWNER ) ) {
```

Comment:
This check is probably best combined with the one above; not much point in having two separate ones for zero return value.

Reply:
Good point.